src/components/Panel/Panel.md

```diff
@@ -0,0 +1,45 @@
+# Panel
+Presents content by sliding over the rest of the application, which is covered by a partially-transparent overlay.
```

Comment:
Panels are interesting in how they compare to other potentially viable containers such as Dialogs or Panes. I'd be a bit more verbose/explicit:
"Containers used in experiences that do not require explicit context for heavy-weight creation/edit/management tasks such as settings, multi-field forms, and permissions. For containers used for complex tasks that requires context, use a separate Pane alongside the existing experience such as a List/Details layout."